addons/test_mail/tests/test_mail_activity.py

```diff
@@ -512,9 +499,79 @@ def test_get_activity_data(self):
                 'ids': set(record_activities.ids),
                 'reporting_date': record_activities[0].date_deadline,
                 'user_assigned_ids': record_activities.user_id.ids,
+                'summaries': [act.summary for act in record_activities],
             })
 
 
+@tests.tagged('mail_activity')
+class TestORM(TestActivityCommon):
```

Comment:
Why those tests ? Isn't it a conflict solving went wrong, as those are now in test_mail_activity_mixin.py ?